### DISCUSSION TITLE
A removed member's email is receiving our Nhost notifications

### DISCUSSION THREAD

**Discussion by Gsl-12** [nhost/nhost#2433] [General]
A member who was removed from the workspace is receiving email notifications from Nhost. Today I received an Nhost error email notification and in the receiving emails I noticed an email that **should not be receiving any information.** That email address may have been the creator of the workspace earlier when that outsourcing company was setting up nhost. How can we remove this email? They are definitely not a member.

**Comment by dbarrosop** [comment]
> may have been the creator of the workspace

Thanks for reaching out. The creator of a workspace is also the owner of such workspace and it certainly receives support emails regarding projects. They also have permissions to create/delete projects and change billing details. If you want to change the owner of the workspace you will need to ask them to email us (support@nhost.io) and request to change the ownership.

**Reply by Gsl-12** [reply]
The workspace says I'm the owner. and they are not a member or the owner currently. They are not in the company. What then? 

Since there is a way to transfer ownership (and there is) then this step should not have been needed. We are no longer in contact with this company/email address owner. Please advise.

**Reply by dbarrosop** [reply]
A workspace can have multiple owners. In any case, if you are an owner, start by sending an email to support@nhost.io from that account indicating the workspace name and we can take it from there.

**Reply by Gsl-12** [reply]
Sounds good

**Comment by nunopato** [comment]
A fix has been released.
